tests/chart_tests/test_postgres.py

```diff
@@ -70,3 +70,64 @@ def test_postgresql_statefulset_with_private_registry_enabled(self, kube_version
                 assert container["image"].startswith(
                     repository
                 ), f"Container named '{name}' does not use registry '{repository}': {container}"
+
```

Comment:
Postgres is missing the `"persistentVolumeClaimRetentionPolicy" not in doc["spec"]` test case